assets/queries/terraform/aws/secure_ciphers_disabled/query.rego

```diff
@@ -9,11 +9,11 @@ CxPolicy[result] {
 		"documentId": input.document[i].id,
 		"searchKey": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version", [name]),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version is TLSv1.1 or TLSv1.2", [name]),
-		"keyActualValue": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version isn't TLSv1.1 or TLSv1.2", [name]),
+		"keyExpectedValue": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version starts with TLSv1.1 or TLSv1.2", [name]),
+		"keyActualValue": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version isn't start with TLSv1.1 or TLSv1.2", [name]),
 	}
 }
 
-checkMinProtocolVersion("TLSv1.1") = true
+checkMinProtocolVersion(startswith("TLSv1.1")) = true
 
-checkMinProtocolVersion("TLSv1.2") = true
+checkMinProtocolVersion(startswith("TLSv1.2")) = true
```

Comment:
Hi @frediana we're very happy that you're always contributing to our project. I want to suggest a few tweaks.

First, let's take a look at this field validation function. I believe this would be the desired behavior.

```suggestion
checkMinProtocolVersion(protocolVersion) {
	startswith(protocolVersion, "TLSv1.1")
} else {
	startswith(protocolVersion, "TLSv1.2")
}
```

Now the unit tests should pass. Furthermore, I think we could add a new negative sample, we should call it `negative2.tf` and rename the old one to `negative1.tf`:

![image](https://user-images.githubusercontent.com/75368139/122892104-0547f600-d33d-11eb-8626-c14af8e8eb67.png)

In this new `negative2.tf` we should add the configuration that we don't want to trigger our result. I suggest copying the contents of `negative1.tf` with this small change:

```
  viewer_certificate {
    cloudfront_default_certificate = true
    minimum_protocol_version       = "TLSv1.2_2019"
  }
```

Reply:
Thank you again for being an active member for our community

Reply:
Thanks for your comment, I will apply update my PR accordingly. I actually struggled a bit to run unit tests for my case only (secure_cyphers_disabled). Somehow, more strangely when running locally a make test, everything is OK. I probbaly doing something wrong, do you have any idea?

Thank you,